must-gather/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM quay.io/openshift/origin-must-gather:latest as builder
```

Comment:
We are probably going to have konflux issues around this image.

Is there an official location for this? registry.redhat.io or registery.redhat.access?

Reply:
I see registry-proxy.engineering.redhat.com/rh-osbs/openshift-ose-must-gather in CNVs custom must-gather Dockerfile.  Tag is a very specific value linked to and OCP build that can only be found in brew.  https://brewweb.engineering.redhat.com/brew/packageinfo?packageID=71510

Reply:
Although EC passed.

Reply:
Yea let's see what happens.